api-lib/helpers/circles.ts

```diff
@@ -0,0 +1,58 @@
+import faker from 'faker';
+import { z } from 'zod';
+
+import { createOrganization } from '.';
+
+const createCircleSchemaInput = z
+  .object({
+    name: z.string().min(3).max(255),
+    organization_id: z.number().int().positive(),
+  })
+  .strict();
+
+type CircleInput = typeof createCircleSchemaInput['_type'];
+
+export function getCircleName() {
+  try {
+    return faker.unique(faker.commerce.department);
+  } catch (_) {
+    return faker.commerce.color();
+  }
+}
+
+export async function createCircle(
+  client: any,
+  object?: Partial<CircleInput>
+): Promise<{ id: number }> {
+  let organizationId = object?.organization_id;
+
+  if (!organizationId) {
+    const organization = await createOrganization(client, { name: 'Org' });
```

Comment:
Can we use `faker` for the org name as well.

Reply:
Definitely, done with https://github.com/coordinape/coordinape/pull/1486/commits/729b3ff990b65da0247d9ee19429bb579379b51e